benefit/data/sheets/CaseEvent/CaseEvent.json

```diff
@@ -1325,7 +1336,7 @@
     "ID": "citizenRequestHearingRecording",
     "Name": "MYA Request hearing recording",
     "Description": "Request a hearing recording",
-    "DisplayOrder": 107,
+    "DisplayOrder": 112,
```

Comment:
All display with order 107 should be updated with same value. Same as first comment